assignment02/index.html

```diff
@@ -1,45 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <title>assignment02</title>
-    <meta charset="utf-8">
-</head>
-
-<body>
-    <!--
-            Allegra French
-            assignment02.html
-            INFO_2340_WW
-            Thoendel
-            12.14.19
-        -->
-
-    <h1>The Wizard of Oz!</h1>
-    <p><strong><span id="female_name_0_1"></span></strong> lived in the midst of the great <strong><span id="state_name"></span></strong> <strong><span
-            id="geographic_feature_1"></span></strong>, with Uncle <strong><span id = "male_name_1"></span></strong>, who was a <strong><span id = "occupation"></span></strong>,
-            and Aunt <strong><span id="female_name_1_1"></span></strong>, who was the farmer's wife. Their house was small, for the lumber to build 
-            it had to be carried by wagon many miles. There were four walls, a floor and a roof, which made one room; and 
-            this room contained a rusty looking <strong><span id = "noun_0"></span></strong>, a <strong><span id = "noun_1"></span></strong> for the dishes, a 
-            table, three or four chairs, and the beds. Uncle <strong><span id = "male_name_2"></span></strong> and <strong><span id = "female_name_1_2"></span></strong>
-            had a big bed in one corner, and <strong><span id = "female_name_0_2"></span></strong> a little bed in another corner. There was no 
-            garret at all, and no cellar-except a small hole dug in the ground, called a cyclone cellar, where the family 
-            could go in case one of those great whirlwinds arose, mighty enough to crush any building in it's path. 
-            It was reached by a trap door in the middle of the <strong><span id = "noun_2"></span></strong>, from which a ladder led down into a small, dark hole.
-    </p>
-    <p>When <strong><span id= "female_name_0_3"></span></strong> stood in the doorway and looked around, she could see nothing but the great
-        <strong><span id = "color_0_1"></span></strong> <strong><span id = "geographic_feature_2"></span></strong> on every side. Not a <strong><span id= "noun_3"></span></strong> nor a 
-        <strong><span id= "noun_4"></span></strong> broke the broad sweep of flat country that reached to the edge of the sky in all directions.
-        The sun had baked the plowed land into a <strong><span id = "color_0_2"></span></strong> mass, with little cracks running through it.
-        Even the grass was not green, for the sun had burned the tops of the long blades until they were the same <strong><span id = "color_0_3"></span></strong> color 
-        to be seen everywhere. Once the house had been painted, but the sun blistered the paint and the rains washed it away, and now the house was as dull 
-        and <strong><span id="color_0_4"></span></strong> as everything else.
-
-    </p>
-    
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-    <script src="js/script.js"></script>

```

Comment:
These script entries should be in the head portion of your HTML document. Not down here.